policy/configurations/comcam.yaml

```diff
@@ -0,0 +1,12 @@
+description: LSST ComCam Parameters
+instrument: lsst.obs.lsst.ComCam
+
+# Pupil inner and outer radius
+pupil:
+  R_inner: 0.0
+  R_outer: 0.335
+
+# Obscuration (inner_radius / outer_radius of field)
+obscuration:
+  R_inner: 2.558
+  R_outer: 4.18
```

Comment:
You may need to add a new line at the end of `yaml` file. This applies to all `yaml` files. BTW, please add the unit 
in the comments as well.